src/Caliburn.Micro.Platform/Platforms/Android/CaliburnApplication.cs

```diff
@@ -131,5 +131,14 @@ protected virtual IEnumerable<object> GetAllInstances(Type service)
         protected virtual void BuildUp(object instance)
         {
         }
+
+        /// <summary>
+        /// Handles exceptions that occur during coroutine execution.
+        /// </summary>
+        /// <param name="sender">The sender of the event.</param>
+        /// <param name="e">The event arguments containing exception details.</param>
+        protected virtual void CoroutineException(object sender, ResultCompletionEventArgs e)
```

Comment:
Will this ever be called? Isn't it missing the following in `Initialize` method?
```c#
            Coroutine.Completed += (s, e) =>
            {
                if (e.Error != null)
                {
                    CoroutineException(s, e);
                }
            };
```
If yes then please also check:

- [src/Caliburn.Micro.Platform/Platforms/Maui/Windows/CaliburnApplication.cs](https://github.com/Caliburn-Micro/Caliburn.Micro/pull/961/files#diff-4465e759a42b36c66175fa8ea4c07f39b5e52ea5916722ea4b4fba7d69ad080e)

- [src/Caliburn.Micro.Platform/Platforms/iOS/CaliburnApplicationDelegate.cs](https://github.com/Caliburn-Micro/Caliburn.Micro/pull/961/files#diff-6f51ee129d2ade8d93cbc6023553855ea0af1909c1a4861a3fb70646345f51ad)